cram_moveit/src/moveit.lisp

```diff
@@ -588,11 +668,54 @@ as only the final configuration IK is generated."
               planning-group pose-stamped
               :allowed-collision-objects allowed-collision-objects
               :plan-only t
+              :path-constraints-msg path-constraints-msg
+              :start-state start-robot-state
               :touch-links touch-links
               :ignore-collisions ignore-collisions
               :max-tilt max-tilt)))))
 
+(defun plan-link-movements (link-name planning-group poses-stamped
```

Comment:
If the path-constraint-msgs parameter is left nil (which is the case for previous CRAM code, since path-constraint-msgs is a new &key parameter added in this PR), then the function takes a branch that is a copy of the previously existing function.

The reason I moved it in the source code is to avoid a function-undefined warning while compiling it (it calls plan-link-movement).